docs/madr/3-handlers.md

```diff
@@ -0,0 +1,112 @@
+# Process Handlers for ANT Registry
+
+- Status: proposed
+- Deciders: [Dylan], [Ariel], [Phil]
+- Date: 2024-07-22
+- Authors: [Atticus], [Dylan]
+
+## Context and Problem Statement
+
+To enable the functionality of the ANT registry as outlined in the [Initial ADR], new process handlers need to be defined and implemented. The main goal is to ensure that the registry is always up to date and can efficiently handle ownership queries and integrations with our current (and future) systems.
+
+## Decision Drivers
+
+The main drivers for this decision revolve around:
+
+- **Efficiency**
+  - Optimized lookups for process IDs and wallet addresses.
+- **Scalability**
+  - Ability to handle increasing numbers of ANTs and associated processes.
+- **Maintainability**
+  - Clear, modular handlers for different operations.
+- **Security**
+  - Consideration for preventing abuse and spam in the future.
+
+## Considered Options
+
+The `State-Notice` and `Access-Control-List` handlers are required for storing the ANT owner, however the `Register` handler was discussed deeply to see if it was necessary at all. The names were decided based on the ANT contracts
```

Comment:
It's unclear to me reading this what the `Register` handler actually does.

Reply:
In fact all the handlers mentioned here are just raised with no previous context. We should explain in the Context section what each of these incoming messages are (or will be in the future).